module/applications/item/ability-use-dialog.mjs

```diff
@@ -58,10 +58,16 @@ export default class AbilityUseDialog extends Dialog {
     const concentrationOptions = this._createConcentrationOptions(item);
     const resourceOptions = this._createResourceOptions(item);
 
+    const slotOptions = this._createSpellSlotOptions(item.actor, item.system.level);
+    if ( item.type === "spell" ) {
+      const slot = slotOptions.find(s => s.key === config.slotLevel) ?? slotOptions.find(s => s.canCast);
+      if ( slot ) item = item.clone({ "system.level": slot.level });
```

Comment:
Why do we need to indiscriminately clone the item if it's a spell?

Reply:
`SummonsData#relevantLevel` fetches the appropriate level from the item's roll data, so we need to have the level be accurate for it to determine the correct levels for summoning in order to display the correct profiles available.

Reply:
OK, that makes sense